impeller/typographer/font_glyph_pair.h

```diff
@@ -20,6 +20,17 @@ struct GlyphProperties {
   Join stroke_join = Join::kMiter;
   Scalar stroke_miter = 4.0;
   bool stroke = false;
+
+  struct Equal {
+    constexpr bool operator()(const impeller::GlyphProperties& lhs,
+                              const impeller::GlyphProperties& rhs) const {
+      return lhs.color.ToARGB() == rhs.color.ToARGB() &&
+             lhs.stroke == rhs.stroke && lhs.stroke_cap == rhs.stroke_cap &&
+             lhs.stroke_join == rhs.stroke_join &&
+             lhs.stroke_miter == rhs.stroke_miter &&
+             lhs.stroke_width == rhs.stroke_width;
```

Comment:
These are floating point, that's okay to use `==`?  Probably, just double checking.

Reply:
I think this is OK? Its not really changed in this PR just moved around.